src/modules/app/actions/sync-blockchain.js

```diff
@@ -40,18 +38,15 @@ export const syncBlockchain = cb => (dispatch, getState) => {
     cb && cb();
   });
 
-  debounce(
```

Comment:
Should we be debouncing this?

Reply:
Dont need it. Sync blockchain is called every block. I recently added it b/c my debug logging suggested it was called twice on occasion. 

What was really going on was augur-node takes longer every second get sync data request. So the responses look like a double tap, but one of them was just lagging.